eva/server/server.py

```diff
@@ -38,6 +38,7 @@ async def start_eva_server(self, host: string, port: int):
         port: port of the server
         """
         logger.info("Start Server")
+        print(f"EVA Server started at host {host} and port {port}", flush=True)
```

Comment:
Is there a reason for using `print` instead of logger here?

Reply:
Logger wasn't displaying anything. Maybe we have to set the level to `info`, wasn't sure if we want to do it.